main-app/client/src/theme.js

```diff
@@ -0,0 +1,62 @@
+import { createMuiTheme } from '@material-ui/core/styles'
+
+function pxToRem(value) {
+  return `${value / 16}rem`
+}
+
+const theme = createMuiTheme({
+  palette: {
+    white: { main: '#FFFFFF' }, //  Button Text, Data Row Background, Top Navigation
+    dataRowBackgroundAlt: { main: '#F4F6F8' },
+    primaryText: { main: '#1F2933' },
+    siteBackground: { main: '#F4F7FB' },
```

Comment:
remove line 13 - 20. We need to create new naming conventions & groupings to better leverage reuse of these components. For example error, success, warning can be in an object like this
`
alerts: {
  error: "#CC3300",
  success: "#339900",
  warning: "#FFCC00"
`